hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/expiry/ExpirySystem.java

```diff
@@ -259,7 +259,7 @@ public void evictExpiredEntries(int percentage, long now, boolean backup) {
 
         scanAndEvictExpiredKeys(maxScannableKeyCount, now, backup);
 
-        accumulateOrSendExpiredKey(null);
+        accumulateOrSendExpiredKey(null, UNSET);
```

Comment:
Not very likely, but this can lead to a hash collision. Isn't it a problem? Wouldn't a nullable `Long` instance fit better here?

Reply:
Actually, it is an ignored param, not used. Seeing method body helps here.

Reply:
Right, I see :+1: Better to name it so. Anyways, the change LGTM. Let me know when you consider it final.

Reply:
renamed it.